### PR TITLE
customer_page: update mobile media

### PR DESCRIPTION
**Pull request by allangrds**
## Description
2018-05-04-174343_3286x1080_scrot

Closes https://github.com/pagarme/mercurio/issues/187 

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/themes/default/customerPage/index.css [hunk 1]
```diff
@@ -41,3 +43,19 @@
...
+
+@media (--media-palm) {
+
+  .inputsContainer {
+    padding-left: 13px;
+    padding-right: 13px;
+  }
+
+  .buttonContainer {
+    margin-top: 22px;
```
Comment (augusto-jm-amaral) [inline src/themes/default/customerPage/index.css]: @allangrds o botão deve estar alinhado com os inputs, talvez tenha que adicionar a mesma `margin` aqui, vamos tentar padronizar isso Hahaha

Comment (allangrds) [reply src/themes/default/customerPage/index.css]: Feito

File: src/themes/default/customerPage/index.css [hunk 2]
```diff
@@ -32,12 +44,18 @@
...
-  display: flex;
-  flex-direction: column;
-  justify-items: flex-end;
-  align-items: flex-end;
-  margin-bottom: var(--spacing-medium);
-  margin-right: var(--spacing-medium);
+@media (--media-palm) {
+
+  .inputsContainer {
+    padding-left: 13px;
```
Comment (augusto-jm-amaral) [inline src/themes/default/customerPage/index.css]: @allangrds podemos remover os paddings, o container pai vai alinhar os inputs.

Comment (allangrds) [reply src/themes/default/customerPage/index.css]: Feito

**Pull request merged**